go-controller/pkg/ovndbmanager/ovndbmanager.go

```diff
@@ -75,14 +73,37 @@ func ensureOvnDBState(db string, kclient kube.Interface, stopCh <-chan struct{})
 	}
 	properties := propertiesForDB(db)
 
+	var dbRetry int32
+
 	for {
 		select {
 		case <-ticker.C:
 			klog.V(5).Infof("Ensure routines for Raft db: %s kicked off by ticker", db)
-			ensureLocalRaftServerID(db)
-			ensureClusterRaftMembership(db, kclient)
+			if err := ensureLocalRaftServerID(db); err != nil {
```

Comment:
I like the fact that the glob variable was removed, but I find this a bit harder to follow plus there's a lot of repetition.
Would it make sense to wrap the error handling logic in a separate function that accepts the pointer to dbRetry?